[ws-daemon] nil pointer dereference

panic: runtime error: invalid memory address or nil pointer dereference

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x992e96]

goroutine 82 [running]:
github.com/gitpod-io/gitpod/supervisor/pkg/gitpod.(*APIoverJSONRPC).GetWorkspace(0x0, 0xdbc200, 0xc000132000, 0xc00003e194, 0x24, 0x0, 0x0, 0x0)
	/tmp/build/components-supervisor--app.744beb1b9b420ffe9fdd5196cb11b22d96c3ab24/_deps/gitpod-core-components-supervisor--app/pkg/gitpod/gitpod-service.go:567 +0x116
github.com/gitpod-io/gitpod/supervisor/pkg/ports.(*ConfigService).Observe.func1(0xc0003b4240, 0xc00070a0c0, 0xc000294960, 0xdbc200, 0xc000132000)
	/tmp/build/components-supervisor--app.744beb1b9b420ffe9fdd5196cb11b22d96c3ab24/_deps/gitpod-core-components-supervisor--app/pkg/ports/ports-config.go:120 +0x154
created by github.com/gitpod-io/gitpod/supervisor/pkg/ports.(*ConfigService).Observe
	/tmp/build/components-supervisor--app.744beb1b9b420ffe9fdd5196cb11b22d96c3ab24/_deps/gitpod-core-components-supervisor--app/pkg/ports/ports-config.go:112 +0xac

--- a/components/ws-daemon/pkg/content/initializer.go
+++ b/components/ws-daemon/pkg/content/initializer.go
@@ -85,9 +85,9 @@ func collectRemoteContent(ctx context.Context, rs storage.DirectAccess, ps stora
 			// no prebuild found - that's fine
 		} else if err != nil {
 			return nil, xerrors.Errorf("cannot find prebuild: %w", err)
+		} else {
+			rc[si.Prebuild.Snapshot] = *info
 		}
-
-		rc[si.Prebuild.Snapshot] = *info
 	}
 
 	return rc, nil

